<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdddelFooterDlg</class>
 <widget class="QDialog" name="AdddelFooterDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>839</width>
    <height>467</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add/Delete Footer</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>271</width>
     <height>391</height>
    </rect>
   </property>
   <property name="title">
    <string>Available Menus</string>
   </property>
   <widget class="QListWidget" name="listWidgetMenu">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>251</width>
      <height>361</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>20</y>
     <width>521</width>
     <height>391</height>
    </rect>
   </property>
   <property name="title">
    <string>Edit Existing Footers</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Name for the Link :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditLink">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>241</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonInsert">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>Insert</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonRemove">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>200</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>Remove</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>10</y>
      <width>211</width>
      <height>371</height>
     </rect>
    </property>
    <property name="title">
     <string>Existing Footers</string>
    </property>
    <widget class="QListWidget" name="listWidgetFooters">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>191</width>
       <height>341</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButtonClose">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>420</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonClose</sender>
   <signal>clicked()</signal>
   <receiver>AdddelFooterDlg</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>753</x>
     <y>379</y>
    </hint>
    <hint type="destinationlabel">
     <x>802</x>
     <y>391</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
